### DISCUSSION TITLE
Welcome to doco-cd Discussions!

### DISCUSSION THREAD

**Discussion by kimdre** [kimdre/doco-cd#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maxexcloo** [comment]
This is awesome! Exactly the tool I’m looking for, appreciate the work you’ve done on this :)

**Comment by asier-paz** [comment]
Hello.

First of all, thanks for this. I was looking for something simple like this for the CD of my apps without having to use Kubernetes.

I followed the docs and I am kind of stucked upping the service. The application fails with this output:
```
doco-cd  | {"time":"2025-05-31T15:14:55.562743074+02:00","level":"info","msg":"starting application","version":"v0.19.1","log_level":"info"}
doco-cd  | {"time":"2025-05-31T15:14:55.570449347+02:00","level":"critical","msg":"failed to retrieve /data mount point for container ","error":"failed to inspect container : invalid container name or ID: value is empty"}
doco-cd exited with code 0
```

I have the `data` volume set up and I even tried with a bind mount with no success:
```
volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - data:/data

volumes:
  data:
```

**Reply by kimdre** [reply]
Hi, can you please tell me your operating system and your docker version (`docker --version`)?

**Reply by kimdre** [reply]
@asier-paz I may have found what's causing the issue and am currently testing a fix (https://github.com/kimdre/doco-cd/pull/411)
The doco-cd container was looking for its container id by checking a file that is only supported on systems with cgroupV1. You probably have a system that uses cgroupV2, so the ID retrieval failed.

**Reply by kimdre** [reply]
@asier-paz Please check the new version [v0.20.0](https://github.com/kimdre/doco-cd/pkgs/container/doco-cd/427937832?tag=0.20.0). Your issue should be fixed now with this.
